src/test/java/millom/sandbox/mapper/NasaMapperTest.java

```diff
@@ -0,0 +1,247 @@
+package millom.sandbox.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import millom.sandbox.pojos.Weather;
+import org.junit.jupiter.api.Test;
+
+public class NasaMapperTest {
+  private NasaMapper nasaMapper = new NasaMapper();
+  public static final String SAMPLE_RESPONSE="{\n"
```

Comment:
This is made it public because using the SAMPLE_RESPONSE in MyClientServiceTest.java and WeatherServiceTest.java classes also.

But now this is moving ti JsonUtility helper class.